sonar-csharp-plugin/src/main/resources/documentation.md

```diff
@@ -33,14 +33,6 @@ As an alternative, an `.editorconfig` file can be used to disable the analysis f
 dotnet_diagnostic.Sxxx.severity = none
 ```
 
-## Improving Performance  
```

Comment:
@tom-howlett-sonarsource Didn't we say that we'll just rephrase it the opposite way? It's an analyzer parameter, so it should be documented. And this is the correct place to do so.

Reply:
No, I don't think so because we've shown the parameter has negligible impact and only now serves as something you might set to false as part of troubleshooting. For that reason, I think having it hidden and just in the community guide is sufficient. I think having it in the docs now just complicates things.